src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -469,5 +479,33 @@ private async Task<ReviewRevisionModel> FindMatchingApprovedRevision(ReviewModel
             }
             return null;
         }
+
+        public async void UpdateReviewBackground()
+        {
+            TelemetryClient telemetryClient = new TelemetryClient(TelemetryConfiguration.CreateDefault());
+
+            // Enabling this only for manual reviews in the beginning to check impact on system performance
+            // We will enable it for all reviews based on the perf details
+            // Automatic reviews are already updated as part of scheduled upload daily
+            var reviews = await _reviewsRepository.GetReviewsAsync(false, "All", false);
+            foreach(var review in reviews.Where(r => IsUpdateAvailable(r)))
+            {
+                var requestTelemetry = new RequestTelemetry { Name = "Updating Review " + review.ReviewId };
+                var operation = telemetryClient.StartOperation(requestTelemetry);
+                try
+                {
+                    await Task.Delay(5000);
```

Comment:
Does the apiview service run on just a single app service instance? If it runs across multiple appservice instances then you might want to consider using a blob lock to coordinate here so you aren't trying to do this across two machines at once.

Reply:
It runs one single instance now.